build.py

```diff
@@ -23,18 +27,263 @@ def wait_until_container_is_ready():
     if(p2.match(matched_line)):
         return True
     return False
-    
-def build_and_deploy():
-    subprocess.run(['docker','build','-f','Dockerfile.x86','-t','docker.io/robertbartlettbaron/acct-mgt.x86','.'])
-    subprocess.run(['docker','push','docker.io/robertbartlettbaron/acct-mgt.x86'])
-    subprocess.run(['oc','-n','acct-mgt','rollout','latest','acct-mgt'])
-    result=subprocess.run(['oc','get','all'],stdout=subprocess.PIPE,stderr=subprocess.STDOUT) 
-    return wait_until_container_is_ready()
+
+def build_and_deploy(docker_image):
+    subprocess.run(['docker','build','-f','Dockerfile.x86','-t',docker_image,'.'])
+    subprocess.run(['docker','push',docker_image])
+    subprocess.run(['oc','-n','acct-mgt-2','rollout','latest','acct-mgt-2'])
+    result=subprocess.run(['oc','get','all'],stdout=subprocess.PIPE,stderr=subprocess.STDOUT)
+    return True
+
+def compare_results(result, pattern):
+    if(result is not None):
+        p1=re.compile(pattern)
+        lines=result.stdout.decode('utf-8').split('\n')
```

Comment:
Consider using `splitlines instead of `split('\n')`. That makes it immediately obvious what you're doing.